source/win-project-properties/App.vue

```diff
@@ -479,6 +497,22 @@ function moveFileUpInExportList (relativeFilePath: string): void {
   projectSettings.value.files.splice(idx, 1)
   projectSettings.value.files.splice(idx - 1, 0, relativeFilePath)
 }
+
+function toggleSelectAllFiles (): void {
+  const allPaths = availableFiles.value.map(file =>
+    pathToUnix(file.path.slice(dirPath.length + 1))
```

Comment:
Given that we need this transformations multiple times in this file by now (I believe there are more that aren't shown in the diff here), I suggest moving this transformation to a computed property. Maybe even `availableFiles`, if we only use that with this transformation?